go/engine/login_username.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
```

Comment:
will any code be deleted as part of this addition, or a bit further down the road?